common/maplib/move-rule.mjs

```diff
@@ -0,0 +1,51 @@
+import {createRequire} from 'module';
+const require = createRequire(import.meta.url);
+
+const config = require('config');
+const movingRequestThreshold = config.get('movingRequestThreshold');
+
+class MoveRule{
```

Comment:
Please add a brief class comment.

Reply:
If this class is simply a collection of these utility functions, consider using an object instead of a class.
We will never instantiate this class.